episodes/episodes.Rmd

```diff
@@ -560,24 +561,6 @@ and right, respectively.
 
 :::::::::::::: callout
 
-### Table alignment best practises
-
-The colons on each side of the `-` in the table dictate how the column is
-aligned. By default, columns are aligned left, but if you add colons on either
-side, that forces the alignment to that side.
-
-In general, most table contents should be left-aligned, with a couple of 
-exceptions:
-
- - numbers should be right aligned
- - symbols, emojis, and other equal-width items may be center-aligned
-
-These conventions make it easer for folks to scan a table and understand its
-contents at a glance. 
-
-::::::::::::::::::::::
-
-
```

Comment:
this transposition missed the opening callout, so it's swallowed the text and left it with a generic callout title.

Because of the way GitHub reviews works, I cannot suggest edits for that opening fence. Could you bring that fence down to after the table? 

![a large callout block that has "Callout" as its title and contains an example and another section called "Table alignment best practices"](https://github.com/carpentries/sandpaper-docs/assets/3639446/01d7ca5b-0c43-4617-8b1f-cd5cea1b115a)

Reply:
Whoops-a-doodle! Think I fixed this now.